process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+* Projects must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+* Projects must have documented, initial project governance
+
+#### Project Process: Sandbox to Incubation and direct entry to Incubation
+
+Projects are required to undergo technical due diligence as a part of the process of entering Incubation, whether as a move from Sandbox to Incubation or enterting Incubation directly. Technical Due Diligence is driven by a TAC or parent WG sponsor with close collaboration of the project. Once the diligence is complete and the proposal made, the Due Diligence document is made available to the community for two weeks to solicit public comment before a TAC vote is called.
+
+
+
+### Graduated
+
+Graduated projects signal the highest level of maturity for an OpenSSF project. Graduated projects have consistent, mature open source development practices with sustained contribution and activity. 
+
+#### Project Reponsibilities
+
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions
+* Follow security best practices, including achieving a [Silver OpenSSF Best Practices badge](https://bestpractices.coreinfrastructure.org/en/criteria)
+* Maintain a point of contact for vulnerability reports and follow coordinated vulnerability disclosure practices
+* Implement, practice, and refine mature software development and release practices, such as adherence to semantic versioning, and having a declared policy for stable releases and backported fixes.
+
+#### Project Support
+* Receives guidance on technical direction from TAC 
+* Receives space for Contributor Summits at OpenSSF bi-annual events
+* Receives OpenSSF Code of Conduct Committee support
+* Receives infrastructure support (details determined by project leads and OpenSSF Budget Committee)
+* Receives support with vulnerability disclosure from the OpenSSF (Vulnerability Disclosure WG)
+* May post project updates and tutorials to the OpenSSF blog
+* May request OpenSSF budget for project improvements such as security audits or time-bound contracting needs
+* May request OpenSSF for sustained maintainer stipends (details determined by OpenSSF and project leads)
+* With additional TAC or WG approval, may fundraise for dedicated project funds, coordinated by the OpenSSF
+* Projects may use the OpenSSF logo to promote their project (in accordance with the trademark guidelines). Projects may be referred to as an "OpenSSF Project" or "OpenSSF $ProjectName."
+
+#### Graduated Project Entry Requirements and Considerations
+
+* Projects must have maintainers with a minimum of three different company affiliations.
```

Comment:
It's okay to have stronger requirements for graduated - still, 3 different companies seems unnecessarily strong. I think 2 would be enough, especially for specialized projects.

Reply:
Respectfully, I disagree. Signaling that a project is "graduated" bears considerable weight as the downstream consumers of the project are substantially more likely to add it to their critical dependency path.

When only two companies support a project with maintainers, if one company were to withdraw support in the future, that would leave the downstream consumers entirely dependent on a single company. Many companies seek out and select open source projects for their perceived neutrality and likely longevity -- precisely what a "graduated" tier is generally perceived to indicate.

Weakening this requirement tips the scales in a very specific way which I am opposed to.